lib/runtime/wasmer/exports_test.go

```diff
@@ -1554,11 +1572,16 @@ func getParachainHostTrie(t *testing.T) *trie.Trie {
 		err := tt.Put(key, value)
 		require.NoError(t, err)
 
-		// TODO: We currently have to manually set the storage expected values for the exports tests.
+		// TODO: We currently have to manually set the storage lookup values for the exports tests.
 		// We could avoid this if we can lookup the value from the `storage` section of the test data easily.
+		// If we use ma[string]Storage as the type for the storage section, the tests fail for some reason.
 		// https://github.com/ChainSafe/gossamer/issues/3405
 		if s.Name == "validationCode" {
-			parachainTestData.Expected["validationCode"] = string(value)
+			parachainTestData.Lookups["validationCode"] = string(value)
+		}
+
+		if s.Name == "currentCodeHash" {
+			parachainTestData.Lookups["currentCodeHash"] = s.Value
```

Comment:
Can we not use `parachainTestData.Expected` here?

Reply:
This still needs work and you can see it in the comment above the code block. For now, I am going to keep it as is since it is cleaner than how it was before.